cafebabel/templates/articles/drafts/create.html

```diff
@@ -1,14 +1,20 @@
 {% extends "base.html" %}
 {% import "articles/macros.html" as macros without context %}
+{% from "_macro-extra-js.html" import extra_js without context %}
 
-{% block extra_js %}<script src={{ url_for('static', filename='js/articles-tags.js') }}></script>{% endblock %}
+{% block extra_js %}
+  {{ extra_js(url_for('static', filename='js/noise.js')) }}
```

Comment:
Would it work if you were writing `{{ extra_js('js/noise.js') }}` here and deport the `url_for` within your macro?
If not there isn't much sense to write `{{ extra_js(` instead of `<script src=`

Reply:
Another option would be to write `{{ extra_js(['js/noise.js', 'js/articles-tags.js', 'js/image-preview.js']) }}` and have a `for…endfor` in the macro.

Reply:
write `{{ extra_js('js/noise.js') }}` doesn't work :-(

Reply:
If it were not working, we could probably keep `{{ extra_js(url_for('static', filename='js/noise.js')) }}` in plain html:`<script src={{url_for('static', filename='js/noise.js')) }}></script>` as we don't gain anything; but I may be missing something here.

Reply:
doesn't work like that: `extra_js('js/noise.js')`. It would work with `extra_js('/static/js/noise.js') though.
Hum you are right, we don't really gain anything here.